hook.php

```diff
@@ -381,9 +381,9 @@ function plugin_glpiinventory_install()
 {
     ini_set("max_execution_time", "0");
 
-    if (basename(filter_input(INPUT_SERVER, "SCRIPT_NAME")) != "cli_install.php") {
+    if (basename(filter_input(INPUT_SERVER, "REQUEST_URI") ?? '') != "cli_install.php") {
```

Comment:
In cli mode, `REQUEST_URI` will probably not be set; and HTTP redirection won't cause effects.

Reply:
```suggestion
    if (basename($_SERVER['SCRIPT_NAME']) != "cli_install.php") {
```